TCL4 Data Management/utm-tcl4-dmp-fet-uas.yaml

```diff
@@ -91,65 +83,171 @@ definitions:
-        minimum: 0.0
-        maximum: 100.0
-        example: 62.88
-      airspeed:
+        example: 15.85        
+          
+      roll_deg:
         description: >-
-          airspeed in ft/sec
+          Roll (deg). Negative roll indicates left.
         type: number
         format: float
-        example: 22.91
-      vertical_speed:
+        example: 15.85        
+        
+      pitch_deg:
         description: >-
-          vertical speed in ft/sec
+          Pitch (deg). Negative pitch indicates nose down.
         type: number
         format: float
-        example: 1.27
-      roll_decimal_degrees:
+        example: 15.85        
+        
+      yaw_deg:
         description: >-
-          roll measured in decimal degrees.
+          Yaw (deg). Zero-degree yaw is North.
         type: number
         format: float
-        example: 29.46
-      pitch_decimal_degrees:
+        example: 15.85        
+        
+      rollRate_degPerSec:
         description: >-
-          pitch measured in decimal degrees.
+          Roll Rate (deg/s). Negative indicates leftward rate.
         type: number
         format: float
-        example: 2.81
-      yaw_decimal_degrees:
+        example: 15.85        
+        
+      pitchRate_degPerSec:
         description: >-
-          yaw measured in decimal degrees.
+          Pitch Rate (deg/s). Negative indicates downward rate.
         type: number
         format: float
-        example: 100.49
-      roll_rate:
+        example: 15.85        
+        
+      yawRate_degPerSec:
         description: >-
-          rate of change of roll angle in degrees/sec.
+          Yaw Rate (deg/s). Negative indicates counter-clockwise rate.
         type: number
         format: float
-        example: 0.37
-      pitch_rate:
+        example: 15.85        
+        
+      hdop_nonDim:
         description: >-
-          rate of change of roll angle in degrees/sec.
+          Number of GPS satellites to which the aircraft has line-of-sight to (whether acquired or not by the GPS receiver).
         type: number
         format: float
-        example: 0.37
-      yaw_rate:
+        example: 15.85        
+        
+      vdop_nonDim:
         description: >-
-          rate of change of roll angle in degrees/sec.
+          VDOP: Vertical dilution of precision of GPS constellation.
         type: number
         format: float
-        example: 0.37
+        example: 15.85        
+        
+      numGpsSatellitesInView_nonDim:
+        description: >-
+          Number of GPS satellites to which the aircraft has line-of-sight to (whether acquired or not by the GPS receiver).
+        type: number
+        format: float
+        example: 15.85        
+        
+      numGpsSat_nonDim:
+        description: >-
+          Number of GPS satellites being tracked by GPS receiver.
+        type: number
+        format: float
+        example: 15.85        
+        
+      batteryVoltage_v:
```

Comment:
What do we do when there are multiple batteries?  This issue is why I designed the CellSample model in https://github.com/nasa/utm-docs/blob/master/TCL4%20Data%20Management/utm-tcl4-dmp-uas-state.yaml.

Reply:
email sent out to the POC in this regard